purchase_order_responsible/models/purchase.py

```diff
@@ -0,0 +1,35 @@
+from odoo import api, fields, models
+
+
+class PurchaseOrder(models.Model):
+    _inherit = "purchase.order"
+
+    # create_uid must mirror the supervisor_id value.
+    # create_uid is a magic field that belongs to the ORM that is not
+    # editable via a form.
+    create_uid = fields.Many2one(
+        comodel_name="res.users", compute="_compute_create_uid"
+    )
+    supervisor_id = fields.Many2one(
```

Comment:
can you add a todo _rename to `responsible_id`_ ?

Reply:
done